datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/all_validations.py

```diff
@@ -64,5 +72,10 @@ def all(ctx):
             echo_info(f'Skipping {validation}')
             continue
         echo_info(f'Executing validation {validation}')
```

Comment:
Unrelated to your change but could this print the name of the validation?
```suggestion
        echo_info(f'Executing validation {validation.name}')
```

This changes from:

```
Executing validation <Command recommended-monitors>
```

to

```
Executing validation recommended-monitors
```